src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -489,5 +489,106 @@ public unsafe bool HasHistoryLog(ClusterOperationContext context, BlittableJsonR
                 return true;
             }
         }
+
+        public static long? TryGetLastRaftIndexForDeleteDatabaseCommandWithoutErrors(ServerStore serverStore, string databaseName, out bool hasExceptions)
```

Comment:
the signature of this method should be
`bool TryGetLastRaftIndexForDeleteDatabaseCommandWithoutErrors(ServerStore serverStore, string databaseName, out bool hasExceptions, out long index)`

Reply:
this way maybe you don't even need the `hasExceptions` ?

Reply:
Done